qutebrowser/config/configtypes.py

```diff
@@ -1223,14 +1224,55 @@ class AcceptCookies(BaseType):
                                ('never', "Don't accept cookies at all."))
 
 
-class ConfirmQuit(BaseType):
+class ConfirmQuit(List):
 
     """Whether to display a confirmation when the window is closed."""
 
+    typestr = 'string-list'
+
     valid_values = ValidValues(('always', "Always show a confirmation."),
                                ('multiple-tabs', "Show a confirmation if "
                                                  "multiple tabs are opened."),
+                               ('downloads', "show a confirmation if downloads "
+                                             "are running"),
                                ('never', "Never show a confirmation."))
+    # Values that can be combined with commas
+    combinable_values = ('multiple-tabs', 'downloads')
+
+    def validate(self, value):
+        values = self.transform(value)
+        # Never can't be set with other options
+        if 'never' in values and len(values) > 1:
+            raise configexc.ValidationError(value, "List cannot contain never!")
+        # Always can't be set with other options
+        elif 'always' in values and len(values) > 1:
+            raise configexc.ValidationError(value,
+                                            "List cannot contain always!")
+        # Values have to be valid
+        elif not set(values).issubset(set(self.valid_values.values)):
+            raise configexc.ValidationError(value, "List contains invalid"
+                                                   " values!")
+        # List can't have duplicates
+        elif len(set(values)) != len(values):
+            raise configexc.ValidationError(value, "List contains duplicate"
+                                                   " values!")
+
+    def complete(self):
+        combinations = []
+        # Generate combinations of the options that can be combined
+        for size in range(2, len(self.combinable_values) + 1):
+            combinations = combinations + list(
+                itertools.combinations(self.combinable_values, size))
+        out = []
+        # Add valid single values
+        for val in self.valid_values:
+            out.append((val, self.valid_values.descriptions[val]))
+        # Add combinations to list of options
+        for val in combinations:
+            desc = ''
+            val = ','.join(val)
+            out.append((val, desc))
```

Comment:
I'd merge those lines into one, probably improves clarity for such a simple thing:

`out.append((','.join(val), ''))`